### DISCUSSION TITLE
Updating Adguard Home LXC

### DISCUSSION THREAD

**Discussion by voiceoverguyuk** [tteck/Proxmox#2827] [Help Needed]
When I update AdGuard Home, I normally simply click on "Update now" and it does it. However, this time, when I click on the button it says "Auto-update failed".  I think it is happening because the container is low on allocated storage space. How can I increase that?

**Comment by voiceoverguyuk** [comment]
I've sorted it. :)

**Reply by soflane** [reply]
how? :)

**Reply by tteck** [reply]
From the Adguard GUI

**Reply by soflane** [reply]
I already tried, but adguard failed to update itself and I got a link to do it manually. I also tried the command from you website to update manually from the lxc container shell but it does nothing
